src/peerconnection.cpp

```diff
@@ -140,6 +140,11 @@ void PeerConnection::setLocalDescription(Description::Type type) {
 	if (!iceTransport)
 		return; // closed
 
+	if (init.iceUfrag && init.icePwd) {
+		PLOG_DEBUG << "Using custom ufrag/pwd";
+		iceTransport->setUfragAndPwd(init.iceUfrag.value(), init.iceUfrag.value());
```

Comment:
Mixup between ufrag and pwd.
```suggestion
		iceTransport->setUfragAndPwd(init.iceUfrag.value(), init.icePwd.value());
```